### PR TITLE
Update bee and use manifest for artifacts

### PR DESCRIPTION
**Pull request by UnityAlex**
Notes:
* Adjusted android NDK to use r19 artifact instead.
* Squashed "Remove Comment" into one commit

### CODE REVIEW

File: external/buildscripts/Build.bee.cs [hunk 1]
```diff
@@ -1,133 +1,81 @@
...
+
+        private static void RegisterCommonArtifacts()
+        {
+            StevedoreArtifacts.Add(new StevedoreArtifact("MonoBleedingEdge"));
+            StevedoreArtifacts.Add(new StevedoreArtifact("reference-assemblies"));
+        }
+
+        private static void RegisterWindowsArtifacts()
+        {
+            StevedoreArtifacts.Add(new StevedoreArtifact("android-ndk-windows-x86_64"));
```
Comment (ashwinimurt) [inline external/buildscripts/Build.bee.cs]: Does this work? the name here does not match the name in the manifest `android-ndk-win`

We have to check/update the build scripts if the ndk directory name has changed

Comment (UnityAlex) [reply external/buildscripts/Build.bee.cs]: It does not I am fixing this currently.

Comment (ashwinimurt) [reply external/buildscripts/Build.bee.cs]: the buildscript needs updated here https://github.com/Unity-Technologies/mono/blob/unity-master/external/buildscripts/build.pl#L671

Comment (UnityAlex) [reply external/buildscripts/Build.bee.cs]: Already was updated earlier (note this PR is for mono upgrade) https://github.com/Unity-Technologies/mono/blob/unity-master-new-unitychanges/external/buildscripts/build.pl#L673

**Review by ashwinimurt — APPROVED** [review]

**Pull request merged**